docs/docs/forms.md

```diff
@@ -2,11 +2,12 @@
 id: forms
 title: Forms
 permalink: docs/forms.html
-prev: state-and-lifecycle.html
-next: lifting-state-up.html
-redirect_from: "/tips/controlled-input-null-value.html"
+prev: 
+next: 
 ---
 
+## Forms
```

Comment:
I intentionally removed this subheader in the past.
If you add it, this is how the page looks:

<img width="729" alt="screen shot 2016-10-25 at 12 07 23" src="https://cloud.githubusercontent.com/assets/810438/19683770/9cc1a44c-9aab-11e6-8839-e905f98b4ec9.png">

The double heading is unnecessary. If you look at other pages, none of the guides have double headings.